README.md
# Transition collective






CHANGELOG.md
# [1.20.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.19.0...v1.20.0) (2022-06-21)


### Features

* contact + styles ([dbe1baa](https://github.com/SocialGouv/transitions-collectives/commit/dbe1baa8609b8e1db07a2eb5999e0b661c7b722f))

# [1.19.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.18.1...v1.19.0) (2022-05-18)


### Features

* update contacts ([b48f504](https://github.com/SocialGouv/transitions-collectives/commit/b48f5042c57f4c9772b66ee406bc6f21bbf43626))

## [1.18.1](https://github.com/SocialGouv/transitions-collectives/compare/v1.18.0...v1.18.1) (2022-05-07)


### Bug Fixes

* **build:** specify node version in docker file ([58a4d05](https://github.com/SocialGouv/transitions-collectives/commit/58a4d05a7a2bff1d5241ebc0e26d35589cb8910e))

# [1.18.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.17.0...v1.18.0) (2022-05-06)


### Features

* update at pro ([45f5edd](https://github.com/SocialGouv/transitions-collectives/commit/45f5edd18a7877970227f4479665be054c0a2447))

# [1.17.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.16.0...v1.17.0) (2022-05-06)


### Features

* update contacts ([2bc7b08](https://github.com/SocialGouv/transitions-collectives/commit/2bc7b08b511bd4a957a49e411c13034368b06ce5))

# [1.16.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.15.2...v1.16.0) (2022-04-16)


### Features

* separate contact types ([90f0d4c](https://github.com/SocialGouv/transitions-collectives/commit/90f0d4c44b93e601f6c0f302dab4da20e4e24a24))

## [1.15.2](https://github.com/SocialGouv/transitions-collectives/compare/v1.15.1...v1.15.2) (2022-04-15)


### Bug Fixes

* prod SSL ([33d9ffe](https://github.com/SocialGouv/transitions-collectives/commit/33d9ffe042522dd9beada7fb7e1580956c836e0d))

## [1.15.1](https://github.com/SocialGouv/transitions-collectives/compare/v1.15.0...v1.15.1) (2022-04-15)


### Bug Fixes

* contacts ([ce692da](https://github.com/SocialGouv/transitions-collectives/commit/ce692da160905c02d4263d287fde8c019353260e))

# [1.15.0](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.16...v1.15.0) (2022-04-15)


### Bug Fixes

* contact search ([a22a5bc](https://github.com/SocialGouv/transitions-collectives/commit/a22a5bc8be7b19a675bc2c21454aa1abbeaf6fb0))


### Features

* add DARP ([841ef61](https://github.com/SocialGouv/transitions-collectives/commit/841ef613d5b412ef5dbf4a15b1600f707bc56cec))
* update contacts ([853fb8c](https://github.com/SocialGouv/transitions-collectives/commit/853fb8c674c9170fe3b15cacf2ddaa4e833fd220))
* update text ([2ec9a5c](https://github.com/SocialGouv/transitions-collectives/commit/2ec9a5c28b109cc39417d796136c085342a14203))

## [1.14.16](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.15...v1.14.16) (2022-03-21)


### Bug Fixes

* **contact:** update contact ([cb0dad1](https://github.com/SocialGouv/transitions-collectives/commit/cb0dad1acd7f946b52a705369c2cea8b27e5b0f5))

## [1.14.15](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.14...v1.14.15) (2022-03-17)


### Bug Fixes

* **ci:** env production ([802c60e](https://github.com/SocialGouv/transitions-collectives/commit/802c60e6861f7d431e24342d17611aac4dd7e076))

## [1.14.14](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.13...v1.14.14) (2022-03-17)


### Bug Fixes

* **ci:** production title ([fbb5877](https://github.com/SocialGouv/transitions-collectives/commit/fbb5877ee3b602ba0c145ad5bb5eb23147d70a50))

## [1.14.13](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.12...v1.14.13) (2022-03-17)


### Bug Fixes

* **ci:**  migrate to kube-workflow ([f5480e5](https://github.com/SocialGouv/transitions-collectives/commit/f5480e53cfa92d9845ca2dc810fd9f6e2ce93652))
* **ci:** migrate to kube-workflow ([ae8a7a2](https://github.com/SocialGouv/transitions-collectives/commit/ae8a7a2d1796cf5bf0fed2a358102509acf5e1b8))
* **ingress:** wip ([6a56e84](https://github.com/SocialGouv/transitions-collectives/commit/6a56e84155d3e7c6e32dc962de0e1cde7d47e30c))
* **probes:** /healthz ([c464528](https://github.com/SocialGouv/transitions-collectives/commit/c464528d2e645f6dc42419da61a01ee9df05e62c))
* **redirect:** add new redirectFrom option ([4666fdc](https://github.com/SocialGouv/transitions-collectives/commit/4666fdce103e9381307d771e509bbbbd9ceae28f))

## [1.14.12](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.11...v1.14.12) (2022-03-17)


### Bug Fixes

* **news:** responsive ui ([e0ed74e](https://github.com/SocialGouv/transitions-collectives/commit/e0ed74e55366f2a3b6229dbdbd9e75570ac2ce2f))

## [1.14.11](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.10...v1.14.11) (2022-03-16)


### Bug Fixes

* akto contact ([82cf19c](https://github.com/SocialGouv/transitions-collectives/commit/82cf19c63bc0015a6a4e44b089b60f70821b5918))
* logo ([cd5acce](https://github.com/SocialGouv/transitions-collectives/commit/cd5acce3062021e7bb380210ee19526efaef9c8f))

## [1.14.10](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.9...v1.14.10) (2022-03-15)


### Bug Fixes

* akto contact ([6c08f5a](https://github.com/SocialGouv/transitions-collectives/commit/6c08f5aa66a039562e9a8a564a822b2880092c38))

## [1.14.9](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.8...v1.14.9) (2022-03-12)


### Bug Fixes

* add question to faq ([ccd85bf](https://github.com/SocialGouv/transitions-collectives/commit/ccd85bf6bec1d3a092832660072e801b4eff7a8b))
* company infography infography ([e3a99b0](https://github.com/SocialGouv/transitions-collectives/commit/e3a99b0530cf99a9e5ac0981aa4b5f27d64bc720))
* do not use map ([ea7dec4](https://github.com/SocialGouv/transitions-collectives/commit/ea7dec446bcb188dc1ed7331da7e7fccd1bda7fd))

## [1.14.8](https://github.com/SocialGouv/transitions-collectives/compare/v1.14.7...v1.14.8) (2022-03-10)


### Bug Fixes


src/lib/data/darps.js
/* eslint-disable sort-keys-fix/sort-keys-fix */
export default [
  {
    departement: "01",
    address: "34 avenue des Belges, 01012 Bourg en Bresse",
    email: "stephane.souques@ain.gouv.fr",
  },
  {
    departement: "02",
    address: "Cité Administrative, 5 Rue Marcel Bleuet, 02000 Laon",
    email: "faycal.auraghi@aisne.gouv.fr",
  },
  {
    departement: "03",
    address: "12 rue de la Fraternité, 03017 Moulins",
    email: "isabelle.pereira@allier.gouv.fr",
  },
  {
    departement: "04",
    address: "",
    email: "christine.didier@alpes-de-haute-provence.gouv.fr",
  },
  {
    departement: "05",
    address: "",
    email: "sandrine.de-chastellier@hautes-alpes.gouv.fr",
  },
  {
    departement: "06",
    address: "",
    email: "cynthia.chu@alpes-maritimes.gouv.fr",
  },
  {
    departement: "07",
    address: "7 boulevard du Lycée, 07007 Privas",
    email: "valerie.veyrenc@ardeche.gouv.fr",
  },
  {
    departement: "08",
    address: "",
    email: "aurelie.roget@ardennes.gouv.fr",
  },
  {
    departement: "09",
    address: "9 rue Paul Delpech, 09000 Foix",
    email: "joan.maissonnier@ariege.gouv.fr",
  },
  {
    departement: "10",
    address: "",
    email: "valerie.roman@aube.gouv.fr",
  },
  {
    departement: "11",
    address: "320 chemin de Manquens, 11890 Carcassonne",
    email: "martial.cholet@aude.gouv.fr",
  },
  {
    departement: "12",
    address: "9 rue de Bruxelles, 12031 Rodez",
    email: "francelyne.calmels@aveyron.gouv.fr",
  },
  {
    departement: "13",
    address: "",
    email: "emilie.desage@bouches-du-rhone.gouv.fr",
  },
  {
    departement: "14",
    address: "",
    email: "sandra.preaud@calvados.gouv.fr",
  },
  {
    departement: "15",
    address: "1 rue de l’Olmet, 15007 Aurillac",
    email: "lucie.estrade@cantal.gouv.fr",
  },
  {
    departement: "16",
    address: "15 rue des frères Lumière, 16001 Angoulême",
    email: "mireille.pochard-plouhinec@charente.gouv.fr",
  },
  {
    departement: "17",
    address: "3 avenue de la Porte Dauphine, 17021 La Rochelle",
    email: "william.vitek@charente-maritime.gouv.fr",
  },
  {
    departement: "18",
    address: "",
    email: "lila.zerzour@cher.gouv.fr",
  },
  {
    departement: "19",
    address: "",
    email: "celine.tissandier@correze.gouv.fr",
  },
  {
    departement: "20",
    address: "",
    email: "gael.mordant-desanti@haute-corse.gouv.fr",
  },
  {
    departement: "21",
    address: "21 bd Voltaire, 21000 Dijon",
    email: "marie.begrand@cote-dor.gouv.fr",
  },
  {
    departement: "22",
    address: "",
    email: "cecile.hinault@cotes-darmor.gouv.fr",
  },
  {
    departement: "23",
    address: "",
    email: "nathalie.juillard@creuse.gouv.fr",
  },
  {
    departement: "24",
    address: "Cité administrative, 24000 Périgueux",
    email: "amelia.chabbert@dordogne.gouv.fr",
  },
  {
    departement: "25",
    address: "5 voie Gisèle Halimi, 25043 Besançon",
    email: "nahim.guemazi@doubs.gouv.fr",
  },
  {
    departement: "26",
    address: "70 avenue de la Marne, 26000 Valence",
    email: "remy.laurent@drome.gouv.fr",
  },
  {
    departement: "27",
    address: "",
    email: "etienne.giraudeau@eure.gouv.fr",
  },
  {
    departement: "28",
    address: "15 place de la République, 28019 Chartres",
    email: "anthony.denne@eure-et-loir.gouv.fr",
  },
  {
    departement: "29",
    address: "",
    email: "katya.bosser@finistere.gouv.fr",
  },
  {
    departement: "30",
    address: "1120 route de Saint-Gilles, 30000 Nîmes",
    email: "didier.pottier@gard.gouv.fr",
  },
  {
    departement: "31",
    address: "5 esplanade Compans Caffarelli, 31080 Toulouse",
    email: "adeline.estay@haute-garonne.gouv.fr",
  },
  {
    departement: "32",
    address: "Cité administrative, Place du Foirail, 32020 Auch",
    email: "anouck.singery@gers.gouv.fr",
  },
  {
    departement: "33",
    address: "",
    email: "anne-camille.lasserre@gironde.gouv.fr",
  },
  {
    departement: "34",
    address: "615 boulevard d'Antigone, 34064 Montpellier",
    email: "bruno.labatut-couairon@herault.gouv.fr",
  },
  {
    departement: "35",
    address: "",
    email: "marie-laure.mazieres-webb@ille-et-vilaine.gouv.fr",
  },
  {
    departement: "36",
    address: "Cité administrative Bertrand, 36020 Chateauroux",
    email: "eve.taurisson@indre.gouv.fr",
  },
  {
    departement: "37",
    address:
      "8 rue Alexander Fleming BP 81656 – 37016 TOURS GRAND TOURS CEDEX 1",
    email: "cecile.hatry@indre-et-loire.gouv.fr",
  },
  {
    departement: "38",
    address: "1 avenue Marie Reynoard, 38029 Grenoble",
    email: "chantal.lucchino@isere.gouv.fr",
  },
  {
    departement: "39",
    address: "",
    email: "natalia.machado@jura.gouv.fr",
  },
  {
    departement: "40",
    address: "",
    email: "marc.neigel@landes.gouv.fr",
  },
  {
    departement: "41",
    address:
      "Centre administratif, Porte B, 34 avenue Maunoury BP 10834-41008 BLOIS CEDEX",
    email: "ddetspp-dev-eco@loir-et-cher.gouv.fr",
  },
  {
    departement: "42",
    address: "11 rue Balaÿ, 42021 Saint Etienne",
    email: "audrey.charret@loire.gouv.fr",
  },
  {
    departement: "43",
    address: "",
    email: "raphael.rouvet@haute-loire.gouv.fr",
  },
  {
    departement: "44",
    address: "",
    email: "suzanne.romestain@loire-atlantique.gouv.fr",
  },
  {
    departement: "45",
    address: "",
    email: "jeremy.romelot@loiret.gouv.fr",
  },
  {
    departement: "46",
    address: "304 rue Victor Hugo, 46 004 Cahors",
    email: "fabienne.sebag@lot.gouv.fr",
  },
  {
    departement: "47",
    address: "",
    email: "sheraze.maazi@lot-et-garonne.gouv.fr",
  },
  {
    departement: "48",
    address: "Cité administrative, 9 rue des Carmes, 48000 Mende",
    email: "segolene.regnier@lozere.gouv.fr",
  },
  {
    departement: "49",
    address: "",
    email: "juliette.mourrieras@maine-et-loire.gouv.fr",
  },
  {
    departement: "50",
    address: "",
    email: "perrine.blay@manche.gouv.fr",
  },
  {
    departement: "51",
    address: "",
    email: "karine.baylac@marne.gouv.fr",
  },
  {
    departement: "52",
    address: "",
    email: "adeline.plantegenet@haute-marne.gouv.fr",
  },
  {
    departement: "53",
    address: "",
    email: "anne-sophie.beaujon@mayenne.gouv.fr",
  },
  {
    departement: "54",
    address: "",
    email: "sebastien.galland@meurthe-et-moselle.gouv.fr",
  },
  {
    departement: "55",
    address: "",
    email: "marie-anne.jouron@meuse.gouv.fr",
  },
  {
    departement: "56",
    address: "",
    email: "ingrid.le-fevre@morbihan.gouv.fr",
  },
  {
    departement: "56",
    address: "",
    email: "ingrid.le-fevre@morbihan.gouv.fr",
  },
  {
    departement: "57",
    address: "",
    email: "christelle.dieudonne@moselle.gouv.fr",
  },
  {
    departement: "58",
    address: "",
    email: "caroline.mansuy@nievre.gouv.fr",
  },
  {
    departement: "59",
    address: "",
    email: "tamara.tagbor@nord.gouv.fr",
  },
  {
    departement: "60",
    address: "",
    email: "isabelle.baptiste@oise.gouv.fr",
  },
  {
    departement: "61",
    address: "",
    email: "estelle.mallet@orne.gouv.fr",
  },
  {
    departement: "62",
    address: "",
    email: "megane.dassonville@pas-de-calais.gouv.fr",
  },
  {
    departement: "63",
    address: "Cité administrative, 2 rue Pélissier, 63034 Clermont Ferrand",
    email: "christelle.rodrigues@puy-de-dome.gouv.fr",
  },
  {
    departement: "64",
    address: "",
    email: "corine.martinez@pyrenees-atlantiques.gouv.fr",
  },
  {
    departement: "65",
    address: "Cité administrative Reffye, Rue Amiral Courbet, 65017 Tarbes",
    email: "arnaud.vignal@hautes-pyrenees.gouv.fr",
  },
  {
    departement: "66",
    address: "76 rue Aristide Brilland, 66026 Perpignan",
    email: "estelle.dujardin@pyrenees-orientales.gouv.fr",
  },
  {
    departement: "67",
    address: "",
    email: "anne-cecile.lauga@bas-rhin.gouv.fr",
  },
  {
    departement: "68",
    address: "",
    email: "sonia.kessour@haut-rhin.gouv.fr",
  },
  {
    departement: "69",
    address: "8/10 rue du Nord, 69625 Villeurbanne",
    email: "mathilde.arnoult@rhone.gouv.fr",
  },
  {
    departement: "70",
    address: "4 place René Hologne, 70006 Vesoul",
    email: "vasilisa.kalentseva@haute-saone.gouv.fr",
  },
  {
    departement: "71",
    address: "",
    email: "anne.olivier@saone-et-loire.gouv.fr",
  },
  {
    departement: "71",
    address: "",
    email: "anne.olivier@saone-et-loire.gouv.fr",
  },
  {
    departement: "72",
    address: "",
    email: "elise.halimi@sarthe.gouv.fr",
  },
  {
    departement: "73",
    address: "321 chemin des Moulins, 73011 Chambery",
    email: "corinne.kauffmann@savoie.gouv.fr",
  },
  {
    departement: "74",
    address: "",
    email: "christine.delbe@haute-savoie.gouv.fr",
  },
  {
    departement: "75",
    address: "",
    email: "marika.demortier@drieets.gouv.fr",
  },
  {
    departement: "76",
    address: "",
    email: "carole.lallouet@seine-maritime.gouv.fr",
  },
  {
    departement: "77",
    address: "",
    email: "amel.amrani@seine-et-marne.gouv.fr",
  },
  {
    departement: "78",
    address: "",
    email: "stanley.guerard1@yvelines.gouv.fr",
  },
  {
    departement: "79",
    address: "4 rue Joseph Cugnot, 79026 Niort",
    email: "benedicte.delbancut@deux-sevres.gouv.fr",
  },
  {
    departement: "80",
    address: "40 rue La Vallée, 80000 Amiens",
    email: "emilie.segard@somme.gouv.fr",
  },
  {
    departement: "81",
    address: "44 Boulevard Maréchal Lannes, 81013 Albi",
    email: "herlene.toulemont@tarn.gouv.fr",
  },
  {
    departement: "82",
    address: "140 avenue Marcel Unal, 82000 Montauban",
    email: "corinne.forest@tarn-et-garonne.gouv.fr",
  },
  {
    departement: "83",
    address: "",
    email: "claire.miguet@var.gouv.fr",
  },
  {
    departement: "84",
    address: "",
    email: "lydia.tordjman@vaucluse.gouv.fr",
  },
  {
    departement: "85",
    address: "",
    email: "christine.dumont@vendee.gouv.fr",
  },
  {
    departement: "86",
    address: "",
    email: "lucile.rengeard@vienne.gouv.fr",
  },
  {
    departement: "87",
    address: "",
    email: "frederic.peytral@haute-vienne.gouv.fr",
  },
  {
    departement: "88",
    address: "",
    email: "laurence.francais@vosges.gouv.fr",
  },
  {
    departement: "89",
    address: "3 rue Jehan Pinard, 89000 Auxerre",
    email: "mourad.youbi@yonne.gouv.fr",
  },
  {
    departement: "90",
    address: "11 rue Legrand, 90016 Belfort",
    email: "sylvie.ducray@territoire-de-belfort.gouv.fr",
  },
  {
    departement: "91",
    address: "",
    email: "annie.lafont@essonne.gouv.fr",
  },
  {
    departement: "94",
    address: "",
    email: "thierry.leroyer@drieets.gouv.fr",
  },
  {
    departement: "94",
    address: "",
    email: "eric.jany@drieets.gouv.fr",
  },
  {
    departement: "95",
    address: "",
    email: "yasmine.destrac@val-doise.gouv.fr",
  },
  {
    departement: "2A",
    address: "",
    email: "valentine.olivieri@dreets.gouv.fr",
  },
  {
    departement: "2B",
    address: "",
    email: "gael.mordant-desanti@haute-corse.gouv.fr",
  },
  {
    departement: "94",
    address: "",
    email: "eric.jany@drieets.gouv.fr",
  },
  {
    departement: "971",
    address: "",
    email: "celine.borel@deets.gouv.fr",
  },
  {
    departement: "972",
    address: "",
    email: "dolor.ravi@deets.gouv.fr",
  },
  {
    departement: "973",
    address: "",
    email: "michel.manguer@dieccte.gouv.fr, sylviane.odang@dieccte.gouv.fr",
  },
  {
    departement: "974",
    address: "",
    email: "eric.bertrand@deets.gouv.fr",
  },
  {
    departement: "976",
    address: "",
    email: "maxime.belkessa@deets.gouv.fr",
  },
]
